test/integration/security/access-sanity.js

```diff
@@ -31,6 +31,7 @@ describe(
     }
```

Comment:
simonbishop@Simons-MacBook-Pro-2 happn-3 % grep -r "users.removePermission" ./test
simonbishop@Simons-MacBook-Pro-2 happn-3 % grep -r "users.upsertPermission" ./test
simonbishop@Simons-MacBook-Pro-2 happn-3 % grep -r "users.listPermissions" ./test 

returned nothing - please can these individual methods be tested

Reply:
upsert and remove permissions are being tested in ./integration/security/user-permissions, lines 1160 and 1179.
Adding a test for list permissions

Reply:
Added test for list permissions at  ./integration/security/user-permissions 1216